checkin.js

```diff
@@ -140,7 +141,7 @@ function checkHealth(event, context, finalCallback) {
 
 function sendCheckin(event, context, aimsC, healthStatus, callback) {
     var checkinValues = {
-        collectorType : 'cwe',
+        collectorType : COLLECTOR_TYPE,
```

Comment:
I think we don't need it anymore as `azcollectSvc` is already initialized with `COLLECTOR_TYPE`

Reply:
+1. But `al-collector-js` is not yet ready for this change. I'll send a follow-up PR.

Reply:
btw if it is a larger piece of work lets maybe postpone it

Reply:
@tomdos please check https://github.com/alertlogic/al-collector-js/pull/5. This need to go first. And I'll address your comment after that.